packages/ui/core-components/src/lib/unsorted/ui/Link.svelte

```diff
@@ -0,0 +1,25 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	import { cn } from '$lib/utils.js';
+	import { toBoolean } from '../../utils.js';
+
+	let className = undefined;
+	export { className as class };
+
+	export let url = '#'; // Destination URL
+	export let label = 'Click here'; // Link text
```

Comment:
Might consider `label` defaulting to `url` when it is unset - I believe that is a common behavior in other systems where you would insert a link